frontend/src/pages/library/LibraryPage.scss

```diff
@@ -18,8 +18,9 @@
     margin-right: 86 * $px;
     font-size: 26 * $px;
     font-weight: 700;
-    line-height: 32 * $px;
+    line-height: 36 * $px;
```

Comment:
Why?

Reply:
to make it the same height as `+ project/+ methodology` buttons to avoid jumping when you switch between tabs